tests/FSharp.Compiler.ComponentTests/ErrorMessages/InvalidLiteralTests.fs

```diff
@@ -0,0 +1,22 @@
+﻿// Copyright (c) Microsoft Corporation.  All Rights Reserved.  See License.txt in the project root for license information.
+
+namespace FSharp.Compiler.ComponentTests.ErrorMessages
+
+open Xunit
+open FSharp.Test.Utilities
+open FSharp.Compiler.SourceCodeServices
+
+
+module ``Invalid literals`` =
```

Comment:
There is another `InvalidNumericLiteralsTests.fs` file, but this doesn't really belong there. I then had to decide between creating a test file for this specific case, or introducing a more general file that could contain other invalid literals. I've opted for the latter.